### PR TITLE
Upgrade to Angular 1.3 Only

### PR DESCRIPTION
**Pull request by erundle**
Changing the project to support angular 1.3+. This required updating ngDocs and fixing the example code since angular did not like the previous "function" definition. 

I made this change since we are looking in the future to add angular-bootstrap 1.3 to this project and angular 1.3 is a required. 

Side Note: I also fixed a previous existing bug with warning/danger icons not showing up for notifications. This was discovered during the test of this update.

### CODE REVIEW

File: dist/angular-patternfly.js [hunk 1]
```diff
@@ -302,7 +304,7 @@ angular.module('patternfly.card').directive('pfCard', function () {
       $scope.updateAvailable();
       $scope.chartConfig.data.columns = [["Used",$scope.used],["Available",$scope.available]];
    };
- }
+  }]);
```
Comment (waldenraines) [inline dist/angular-patternfly.js]: Nitpick:  these closing brackets and the parenthesis don't line up with the `angular.module` above.  This is also visible in the docs examples.

**Comment by dtaylor113** [comment]
LGTM

**Comment by waldenraines** [comment]
Looks good to me minus the aforementioned nitpick.

**Comment by jeff-phillips-18** [comment]
+1

File: dist/angular-patternfly.js (hunk 1 above)
Comment (erundle) [inline dist/angular-patternfly.js]: fixed

**Comment by waldenraines** [comment]
LGTM :+1:

**Pull request merged**